setup.py

```diff
@@ -1,3 +1,20 @@
+# (C) Copyright ${year} core.ai (https://core.ai/) 
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Contributors: Author 1, author 2...
```

Comment:
you could add your name here or could specify `contributors: your name and others[see commit log]`

Reply:
Done